configure.ac

```diff
@@ -3535,6 +3535,7 @@ AC_CHECK_FUNCS([fnmatch \
   setlocale \
   setmode \
   setrlimit \
+  snprintf \
```

Comment:
Is it sufficient to have the function here or do we want it to have conforming behavior? Early MSVC and other implementations have issues related to null-termination and return values.

Reply:
Hm, complicated. Can the autotools build even be used to compile with MSVC though?

Reply:
No, but CMake can. And it's not only MSVC, unfortunately. At least also HP, according to https://demin.ws/blog/english/2013/01/28/use-snprintf-on-different-platforms/.

Reply:
The HPUX behavior shown there actually seems fine for our use. Of that listing, the worst one is Windows 7, Visual Studio 12 since it doesn't zero terminate when capped. All the others do.

So no, with MSVC it seems just detecting it is not enough...